website/static/js/pages/prereg-admin-page.js

```diff
@@ -38,10 +41,9 @@ function adminView(data) {
 
     // variables for editing items in row
     self.edit = ko.observable(false);
-    self.item = ko.observable();
     self.commentsSent = ko.observable('no');
```

Comment:
These observables are relevant on a draft-by-draft basis. They should be part of the draft view model.